packages/quantic/force-app/main/default/lwc/quanticHeadlessLoader/__tests__/quanticHeadlessLoader.test.js

```diff
@@ -392,4 +393,23 @@ describe('c/quanticHeadlessLoader', () => {
       });
     });
   });
+
+  describe('destroyEngine', () => {
```

Comment:
Should you have a test to make sure the destroyEngine doesn't destroy another existing engine?